app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1256,16 +1256,28 @@
                             <show_in_website>0</show_in_website>
                             <show_in_store>0</show_in_store>
                         </extensions_compatibility_mode>
-                        <crate_admin_user_notification translate="label comment">
+                        <create_admin_user_notification translate="label comment">
                             <label>New Admin User Create Notification</label>
-                            <comment>This setting enable notification when new admin user created.</comment>
+                            <comment>This setting enables notification when new admin user is created.</comment>
```

Comment:
This is ok to change.

Reply:
Before changing the comment, please check if there is a translation for that phrase. If exists you should change only the CSV file.

Reply:
yep, needs to be changed in Mage_Core.csv instead of here, so that we don't break language packs